### ISSUE TITLE
Using a date later than 30th December 2020 as maxDate disables all days

### ISSUE THREAD

**Issue by Kev1n337** [maurovieirareis/hello-week#50]
maxDate does not work for days later than 31st December 2020. When this date is set as maxDate, all dates are disabled, not only the ones below min and above max.

Reproduction:
```
// Does not work
let helloweek = new HelloWeek({
      selector: '#calendar',
      minDate: new Date(),
      maxDate: new Date('2020-12-31'), // and all dates after this one
});

// Works fine
let helloweek = new HelloWeek({
      selector: '#calendar',
      minDate: new Date(),
      maxDate: new Date('2020-12-30'), // and all dates before this one
});
```

**Comment by mauroreisvieira** [comment]
Hello @Kev1n337 I don't know if I understand well the issue that you are reporting.

So you are saying that when we do that:

```js
new HelloWeek({
      selector: '#calendar',
      minDate: new Date(),
      maxDate: new Date('2020-12-31')
})
```
the December 31 should't be include?
If so, I don't consider that an issue, because it's always depending the situation.

Look to this exemple: https://hello-week.now.sh/#/docs/v2/min-and-max

**Comment by amilham** [comment]
I've run into this as well. The problem is that if `maxDate` is set to 2020-12-31 or later, *all* dates get grayed out and unselectable. This is definitely not correct behavior. What's more, I was finding that if I've got `minDate` set to `new Date()` and no max date, then dates are enabled only up to January 1. I had to remove both the `minDate` and `maxDate` settings to get the calendar working for dates past January 1, 2021.
